operations/filter_kgraph_top_n.yml

```diff
@@ -0,0 +1,51 @@
+id: filter_kgraph_top_n
+subclass_of: filter_kgraph
+name: filter kgraph (top n)
+description: This operation removes kgraph edges that have attribute values are below/above the top/bottom n values.
+examples:
+- input: fill_and_filter/messages/05_filtered_kgraph_stat_input.json
+  output: fill_and_filter/messages/08_filtered_kgraph_top_n_output.json
+  parameters:
+    edge_attribute: normalized_google_distance 
+    threshold: 3
+    keep_top_or_bottom: bottom
+    qnode_keys: 
+      - drug
+input_requirements: []
+output_guarantees:
+- no edges in the kgraph which have attribute values are below/above the top/bottom n values.
+allowed_changes:
+- remove knodes
+- remove kedges
+parameters:
+  type: object
+  properties:
+    edge_attribute:
+      description: The name of the edge attribute to filter on.
+      type: string
+      example: normalized_google_distance
+    max_edges:
+      description: The number of edges to keep.
+      type: integer
+      example: 10
+      minimum: 0
+      default: 50
+    keep_top_or_bottom:
+      description: Indicate whether or not the the top or bottom n values should be kept.
+      type: string
+      enum:
+      - top
+      - bottom
+      default: top
+    qedge_keys:
+      description: This indicates if you only want to filter on specific edge_keys. If not provided or empty, all edges will be filtered on.
+      type: array
+      example: [e01]
+      default: []
```

Comment:
I suggest we just remove the default, but keep the documentation about what happens if they key is not present.